client/platform/desktop/frontend/components/Settings.vue

```diff
@@ -69,23 +75,26 @@ export default defineComponent({
     <v-container>
       <v-card>
         <v-card-title>Settings</v-card-title>
+
         <v-card-text>
-          <v-row>
+          <v-row class="mb-6">
             <v-col cols="9">
               <v-text-field
                 v-model="localSettings.viamePath"
                 label="VIAME Install Base Path"
                 hint="download from https://viametoolkit.com"
                 dense
+                :disabled="!!(localSettings.overrides && localSettings.overrides.viamePath)"
                 persistent-hint
               />
             </v-col>
+
             <v-col cols="3">
               <v-btn
                 large
                 block
                 color="primary"
-                class="mb-6"
+                :disabled="!!(localSettings.overrides && localSettings.overrides.viamePath)"
```

Comment:
I think these two need some explicit text stating why they are disabled.  Something about the environment variable name being set and using that instead of manually allowing changing.

Reply:
Good point. I'll add an info box saying the environment exists and changes are being locked.